src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,7 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(ONWIN64EXCEPTIONS(FaultingExceptionFrame *pfef))
```

Comment:
The ONWIN64EXCEPTIONS is not needed here, as we now have the WIN64EXCEPTIONS always on for Linux x86.